.github/workflows/release.yml

```diff
@@ -331,65 +315,25 @@ jobs:
           rm languages/python/oso/README.md
           cp README.md languages/python/oso/README.md
       - name: Build wheels
-        run: |
-          python -m cibuildwheel --output-dir wheelhouse
-        working-directory: languages/python/oso
+        uses: pypa/cibuildwheel@v2.8.1
+        env:
+          # Skip Python 2.7 and Python 3.5
+          CIBW_SKIP: "cp27-* cp35-* pp27-*"
+          # 64-bit builds only
+          CIBW_BUILD: "*64"
+          # Used in build.py to find right files
+          CIBW_ENVIRONMENT: OSO_ENV=CI
+        with:
+          package-dir: languages/python/oso
       - uses: actions/upload-artifact@v2
         with:
           name: wheel
-          path: languages/python/oso/wheelhouse/*.whl
+          path: ./wheelhouse/*.whl
 
   build_macos_wheels:
     name: Build wheels on MacOS
-    # Build on 10.15 so we can still build a 3.6 wheel.
-    # May have to drop once github actions switches to macos-11 by default
-    # because 3.6 isn't supported on 11.
-    runs-on: macos-10.15
-    needs: [version, macos_libs]
-    env:
-      # Skip Python 2.7 and Python 3.5
-      CIBW_SKIP: "cp27-* cp35-* pp27-*"
-      # 64-bit builds only
-      CIBW_BUILD: "*64"
-      # Used in build.py to find right files
-      CIBW_ENVIRONMENT: OSO_ENV=CI
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        name: Install Python
-        with:
-          python-version: "3.7"
-      - name: Install cibuildwheel
-        run: |
-          python -m pip install cibuildwheel==2.2.2
-      - name: Download static libs
-        uses: actions/download-artifact@v1
-        with:
-          name: oso_static_library
-      - name: Copy static library.
-        run: |
-          mkdir -p languages/python/oso/native
-          cp -r oso_static_library/libpolar-macOS.a languages/python/oso/native/libpolar.a
-          cp -r oso_static_library/polar.h languages/python/oso/native/polar.h
-      - name: Copy in readme
-        run: |
-          rm languages/python/oso/README.md
-          cp README.md languages/python/oso/README.md
-      - name: Build wheels
-        run: |
-          python -m cibuildwheel --output-dir wheelhouse
-        working-directory: languages/python/oso
-      - uses: actions/upload-artifact@v2
-        with:
-          name: wheel
-          path: languages/python/oso/wheelhouse/*.whl
-
-  build_macos_arm_wheels:
```

Comment:
No longer needed. cibuildwheel itself can cross-compile wheels, and by using the fat static lib it'll work for both.